src/libraries/sendtohelix-browser.targets

```diff
@@ -243,6 +243,9 @@
         <Command>$(HelixCommand)</Command>
         <Timeout>$(_workItemTimeout)</Timeout>
         <Timeout Condition="'$(Scenario)' == 'WasmTestOnNodeJS' and '%(FileName)' == 'System.Net.Http.Functional.Tests'">01:20:00</Timeout>
+        <!-- VSF is emulated on the UI thread and all calls are slow because they are marshaled -->
+        <Timeout Condition="'%(FileName)' == 'System.IO.FileSystem.Tests' and '$(WasmEnableThreads)' == 'true'">01:20:00</Timeout>
+        <Timeout Condition="'%(FileName)' == 'System.Private.Xml.Tests' and '$(WasmEnableThreads)' == 'true'">01:20:00</Timeout>
```

Comment:
curious why we need this here, is the one in the .csproj not enough?

Reply:
they are separate msbuild run. This runs once for all assemblies.